Swift/Tests/MessageEndpointTest.swift

```diff
@@ -136,7 +136,8 @@ MultipeerConnectionDelegate {
         browser!.delegate = self
         browser!.startBrowsingForPeers()
         
-        self.wait(for: [clientConnected!, serverConnected!], timeout: 10.0)
+        // cool down period(disconnected to next connected state), is taking around 4-10secs
+        self.wait(for: [clientConnected!, serverConnected!], timeout: 30.0)
```

Comment:
https://issues.couchbase.com/browse/CBL-2959

Reply:
What is it talking so long to be disconnected?

Reply:
it is trying to connect, connection gets refused, connection resets and it repeats again.

Reply:
If it is not supposed to get refused, we should investigate and fix it.

Reply:
The MesseageEndpoint tests seems to be a liability. I think we need to do something about it. Seem like some random things keep happening.

Reply:
Please create a ticket for this if you haven't done.

Reply:
Have added a comment on [CBL-2959](https://issues.couchbase.com/browse/CBL-2959)